spec/onlyoffice_logger_helper_spec.rb

```diff
@@ -17,4 +17,24 @@
     expect { described_class.log('Hello', 31) }
       .to output(/31/).to_stdout
   end
+
+  it 'Output have code green color if it specified' do
+    expect { described_class.green_log('Hello') }
```

Comment:
Minor refactor - I think `'Hello'` string can be store in

`let(:simple_string) { 'Hello' }`

to reduce code duplication